.jenkins/Jenkinsfile-wheel-builder

```diff
@@ -89,13 +89,39 @@ def build(version, label, imageName) {
                 }
                 sh """#!/usr/bin/env bash
                     set -x -e
+                    LIBFFI_SHA256="d06ebb8e1d9a22d19e38d63fdb83954253f39bedc5d46232a05645685722ca37"
+                    LIBFFI_VERSION="3.2.1"
+
+                    function check_sha256sum {
+                        local fname=\$1
+                        local sha256=\$2
+                        echo "\${sha256}  \${fname}" > \${fname}.sha256
+                        sha256sum -c \${fname}.sha256
+                    }
                     # Because we are doing this as root in the container, but we write to a mounted dir that is outside the container
                     # we need to make sure we set these files writable such that the jenkins user can delete them afterwards
                     mkdir -p tmpwheelhouse
                     mkdir -p wheelhouse
                     chmod -R 777 tmpwheelhouse
                     chmod -R 777 wheelhouse
 
+                    # We need libffi because pip wheel won't use wheels at all
+                    curl -#O https://mirrors.ocf.berkeley.edu/debian/pool/main/libf/libffi/libffi_\${  LIBFFI_VERSION}.orig.tar.gz
```

Comment:
the space after the `${` can't be right?

Reply:
Nope, that's a copy paste error due to going through two diff vim processes so I couldn't use the yank buffer. I accept no responsibility for my mistakes

Reply:
have you considered becoming a 🐴